<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post3dWindowNodeGroupArrowFaceSettingWidget</class>
 <widget class="QWidget" name="Post3dWindowNodeGroupArrowFaceSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="directionGroupBox">
       <property name="title">
        <string>Direction</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="iRadioButton">
          <property name="text">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="jRadioButton">
          <property name="text">
           <string>J</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="kRadioButton">
          <property name="text">
           <string>K</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>202</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="rangeGroupBox">
       <property name="title">
        <string>Range</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>I Min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SliderWithValue" name="iMinSlider">
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>I Max</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SliderWithValue" name="iMaxSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>J Min</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="SliderWithValue" name="jMinSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>J Max</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="SliderWithValue" name="jMaxSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>K Min</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="SliderWithValue" name="kMinSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>K Max</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="SliderWithValue" name="kMaxSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enabledCheckBox">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>9</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="samplingGroupBox">
       <property name="title">
        <string>Sampling</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="samplingAllRadioButton">
          <property name="text">
           <string>All vertices</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="samplingSkipRadioButton">
          <property name="text">
           <string>Sampling rate:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="iSamplingRateLabel">
              <property name="text">
               <string>I-direction</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="iSamplingRateSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="jSamplingRateLabel">
              <property name="text">
               <string>J-direction</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="jSamplingRateSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="kSamplingRateLabel">
              <property name="text">
               <string>K-direction</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="kSamplingRateSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="colorGroupBox">
       <property name="title">
        <string>Color</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="specificRadioButton">
          <property name="text">
           <string>Custom color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="ColorEditWidget" name="colorEditWidget"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="scalarComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="scalarRadioButton">
          <property name="text">
           <string>By scalar value</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Arrow Shape</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="arrowSizeLabel">
            <property name="text">
             <string>Arrow size:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="arrowSizeSpinBox">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lineWidthLabel">
            <property name="text">
             <string>Line width:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="lineWidthSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>11</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>157</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>9</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliderWithValue</class>
   <extends>QSlider</extends>
   <header>guibase/widget/sliderwithvalue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>samplingSkipRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>iSamplingRateSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingSkipRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>jSamplingRateSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingSkipRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>kSamplingRateSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>directionGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>samplingGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>scalarComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specificRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorEditWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
